src/StockportWebapp/Views/stockportgov/Comms/Index.cshtml

```diff
@@ -75,19 +74,26 @@
             ViewData["SignUpAlertsTopicIdTopicId"] = Model.Homepage.EmailAlertsTopicId;
         }
         <partial name="../Shared/SignUpAlerts.cshtml" />
-        <h2 class="h3">Useful links</h2>
-        <ul class="useful-links">
-            @foreach (var link in Model.Homepage.UsefullLinks)
-            {
-                <a href="@link.Url" title="@link.Text">
+
+        @if (Model.Homepage.UsefullLinks.Any())
```

Comment:
As we are cleaning this up can we rename the prop to be UsefulLinks to match what is in ContentAPI.